src/Social/Services/Messages.php

```diff
@@ -42,12 +56,15 @@ public static function getMessage(string $uuid): MessagesInterface
      */
     public static function create(UserInterface $user, string $verb, array $message = [], ?MessageableEntityInterface $object = null): MessagesInterface
     {
+        $random = new Random();
         $newMessage = new MessagesModel();
+        $newMessage->uuid = $random->uuid();
```

Comment:
@rwhite27 I made this so use it https://github.com/gewaer/gewaer-api/blob/development/library/Models/Leads.php#L80to avoid this need ;)